### PR TITLE
[FIX] Some stylistic comments to #1 and nit picks

### PR DESCRIPTION
**Pull request by oesteban**

### CODE REVIEW

**Comment by oesteban** [comment]
@satra I can't assign @djarecka as a reviewer

File: engine/auxiliary.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 import inspect, os
 import logging
 logger = logging.getLogger('nipype.workflow')
-from nipype import Node
+from .node import Node
```
Comment (djarecka) [inline engine/auxiliary.py]: This is the `Node` current nipype and I don't have `node.py` 

This is only used to my wrapper for the current interfaces. I'll work now to connect the current interfaces directly, but for now it `CurrentInterface` is using nipype.Node

File: pydra/engine/node.py [hunk 2]
```diff
@@ -1,39 +1,55 @@
...
+
+        Parameters
+        ----------
+
+        name : str
+            Unique name of this node
+        mapper : str or (list or tuple of (str or mappers))
+            Whether inputs should be mapped at run time
+        inputs : str or list of str
+            States this node's input names
```
Comment (djarecka) [inline pydra/engine/node.py]: this is now dictionary, but I can update it with other things

**Comment by djarecka** [comment]
@oesteban - I've fixed all tests and created a PR to your fork

**Comment by effigies** [comment]
Merged master to get coverage reports.

**Pull request merged**